plugin/trino-bigquery/src/test/java/io/trino/plugin/bigquery/TestBigQueryIntegrationSmokeTest.java

```diff
@@ -403,6 +405,21 @@ public void testTimeType()
         onBigQuery("DROP TABLE " + tableName);
     }
 
+    @Test
+    public void testDatetimeType()
+    {
+        try (TestTable table = new TestTable(
+                bigQuerySqlExecutor,
+                "test.test_datetime_type",
+                "(a DATETIME)",
+                ImmutableList.of("'2021-08-27 12:34:56.123'", "'2022-09-28 01:02:03.987'"))) {
+            assertThat(query("SELECT a FROM " + table.getName()))
+                    .matches("VALUES TIMESTAMP '2021-08-27 12:34:56.123', TIMESTAMP '2022-09-28 01:02:03.987'");
+            assertThat(query("SELECT a FROM " + table.getName() + " WHERE a = TIMESTAMP '2021-08-27 12:34:56.123'"))
```

Comment:
any chance to have `io.trino.testing.AbstractTestDistributedQueries#testDataMappingSmokeTest` run for BQ?
it would probably cover this

Reply:
BQ connector doesn't support writes

Reply:
Could it support writes for test purposes?

Reply:
Storage write API is in the beta preview right now